2020-pip/RFP.md

```diff
@@ -0,0 +1,278 @@
+- Help new contributors develop into continuing contributors
+- Help existing contributors grow into co-maintainers
+
+
+June:
+- Review and respond to code contributions and new issues: 40 hours (probably 4 weeks at 10 hours/week)
+
+July:
+- _vacation, unpaid_
+
+August:
+- Review and respond to code contributions and new issues: 40 hours (probably 4 weeks at 10 hours/week)
+
+September:
+- Review and respond to code contributions and new issues: 40 hours (probably 4 weeks at 10 hours/week)
+
+October:
+- Review and respond to code contributions and new issues, and possibly participate in user experience training: 40 hours (probably 4 weeks at 10 hours/week)
+
+November:
+- Review and respond to code contributions and new issues: 30 hours (probably 3 weeks at 10 hours/week)
+
+December:
+- Review and respond to code contributions and new issues: 20 hours (probably 2 weeks at 10 hours/week)
+
+## Expectations and Requirements
+
+### Project Timeline
+
+This project is intended to be completed over three phases, totalling a little over one year, starting mid-December 2019 or early January 2020.
+
+Role 1 and Role 2 have different timelines; see those roles for details.
+
+### Technology
+
+The [pip codebase](https://github.com/pypa/pip) is licensed under the [MIT License](https://github.com/pypa/pip/blob/master/LICENSE.txt). All work submitted or dependencies added must be compliant with this license.
+
+The backend codebase is in Python. The pip documentation is built using reStructuredText and Sphinx.
+
+Potential proposers should be comfortable with Python, and will have support from an additional contractor focused on user interface and user experience design to advise on command line experience improvements.
+
+For Role 1 (Senior developer): Familiarity and expertise with the pip codebase, or with a Python packaging/distribution toolchain codebase, is strongly desired.
+
+For Role 2: Familiarity and expertise with all technologies is not required. Strong Python skills and experience are a must, though. Experience with some automated testing tools (not necessarily the ones we use) are strongly desired.
+
+#### Specific Technologies Used
+
+pip is a Python program that runs on client computers as part of [the Python packaging and distribution toolchain](packaging.python.org/). Understanding of and familiarity with how Python packaging works, how packages are built and installed, and so on, will be helpful.
+
+##### pip codebase
+
+For the best primer, see the [developer documentation](https://pip.pypa.io) for pip.
+
+You can also see the [complete codebase](https://github.com/pypa/pip) on GitHub.
+
+ - [Python](https://www.python.org/) 2.7 and 3.5+ on Linux, Mac OS, and Windows
+ - [pytest](http://pytest.readthedocs.io/)
+ - [tox](https://tox.readthedocs.io/en/latest/)
+ - [Sphinx](http://www.sphinx-doc.org/)
+
+##### Backend and Testing Infrastructure
```

Comment:
I guess this "Backend" is also extraneous?

```suggestion
##### Testing Infrastructure
```

Reply:
Yes, thanks, fixed.